roaring/roaring.go

```diff
@@ -1563,6 +1563,127 @@ func differenceBitmapBitmap(a, b *container) *container {
 	}
 	return output
 }
+func (b *Bitmap) Xor(other *Bitmap) *Bitmap {
```

Comment:
this may seem picky, but i would move the `Xor` function up to be with the other functions on the `(b *Bitmap)` struct, and i would move the remaining functions directly after `differenceBitmapBitmap` in the file just to be consistent with the existing organization of the file.